src/index.js

```diff
@@ -2,16 +2,23 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Navbar from "./components/js/Navbar.js"
+import Home from "./components/js/Home.js"
+
 
 ReactDOM.render(
   <React.StrictMode>
+    <Navbar />
+    <Home />
     <App />
+
```

Comment:
You should put the `Nav`, `Home`, and all other components inside the `App.js` component and not in `index.js`